### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rossmoody**
Having just gone through the installation instructions on both my work and home comp, thought I was in a semi unique position to contribute some general updates to the `README` setup for places I got hung up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,34 @@
...
-
 [![Build Status](https://travis-ci.org/brave-intl/publishers.svg?branch=master)](https://travis-ci.org/brave-intl/publishers)
 
-## Quick start
-
-### Setup
+## :wrench: Setup
 
-These steps presume you are using OSX and [Homebrew](https://brew.sh/).
+Follow these steps to setup the App for [publishers.brave.com](https://publishers.brave.com). This guide presumes you are using OSX and [Homebrew](https://brew.sh/).
```
Comment (corymcdonald) [inline README.md]: Url should be https://publishers.basicattentiontoken.org/

File: README.md [hunk 2]
```diff
@@ -1,40 +1,34 @@
-App for [publishers.brave.com](https://publishers.brave.com).
-
 [![Build Status](https://travis-ci.org/brave-intl/publishers.svg?branch=master)](https://travis-ci.org/brave-intl/publishers)
 
-## Quick start
-
-### Setup
+## :wrench: Setup
```
Comment (corymcdonald) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (rossmoody) [reply README.md]: I was so curious about this, it seems to redirect but regardless I'll fix.

**Comment by rossmoody** [comment]
@nvonpentz all fixed up!

**Pull request merged**